src/kimmdy/runmanager.py

```diff
@@ -386,6 +385,12 @@ def _setup(self, files: TaskFiles) -> TaskFiles:
         logger.info("Start setup task")
         self.state = State.SETUP
         logger.info("Writing initial topology after parsing")
+
+        if self.config.parameterize_at_setup:
+            focus_nrs = set([atom.nr for atom in self.top.atoms.values()])
```

Comment:
Shouldn't this just be the keys of `self.top.atoms`? This wouldn't need iteration then.

Reply:
yeah I thought the same but remembered having problems with dict keys and the actual atom numbers differing. This would of course be a serious problem anyway so I shouldn't try to work around this problem, I guess...